cdk/visit/lambda_code/register_user/register_user.py

```diff
@@ -21,27 +21,27 @@ def __init__(self, table):
             # Get the service resource.
             dynamodb = boto3.resource('dynamodb')
             # Get the table name.
-            TABLE_NAME = os.environ["TABLE_NAME"]
+            TABLE_NAME = os.environ["USERS_TABLE_NAME"]
             # Get table objects
             self.users = dynamodb.Table(TABLE_NAME)
         else:
             self.users = table
 
-    def addUserInfo(self, user_info):
+    def add_user_info(self, user_info):
         # Get the current date at which the user registers.
         timestamp = datetime.datetime.now()
 
         response = self.users.put_item(
             Item={
-                'PK': user_info['username'],
-                'SK': str(timestamp),
-                'firstName': user_info['firstName'],
-                'lastName': user_info['lastName'],
-                'Gender': user_info['Gender'],
-                'DOB': user_info['DOB'],
-                'Grad_date': user_info['Grad_Date'],
-                'Major': user_info['Major'],
-                'Minor': user_info['Minor']
+                'username': user_info['username'],
+                'register_time': str(timestamp),
+                'first_name': user_info['first_name'],
+                'last_name': user_info['last_name'],
+                'gender': user_info['gender'],
+                'date_of_birth': user_info['date_of_birth'],
+                'grad_date': user_info['grad_date'],
```

Comment:
should this be grad_year or grad_semester? "grad_date" is fine as well but we just want to make sure that the actual values that we store are the semester + year

Reply:
So, right now that is dependent on #64. Currently, the mainline branch just sends the date combined, but once #64 is merged we will then need multiple fields.